apps/console/src/features/authorization/models/authorization.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * Interface to store authorization details.
+ */
+export interface AuthorizationInterface {
```

Comment:
Lets move this to `[use-authorization.t](https://github.com/wso2/identity-apps/pull/4368/files#diff-12e93760b28bfb2e85c786ed5f9b351fd41477e763024337bcb3846a11e9ba27)` as `UseAuthorizationInterface`.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/4368/commits/247aec5782d7ace24b15cfe9e857299100328b4a